common/sass/styles/overrides/react-select.scss

```diff
@@ -0,0 +1,62 @@
+// This syntax is necessary to override css styles in react-select
+.Select {
+  font-size: 1rem;
+  .Select-control {
+    box-sizing: content-box;
+    font-weight: 400;
+    border-radius: 0px;
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+    border: 1px solid #ececec;
+    transition: all 120ms;
+    &:hover {
+      box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+      border: 1px solid #ececec;
+    }
+    .Select-value {
+      padding-left: 16px;
+      padding-right: 16px;
+    }
+  }
+  .Select-menu-outer {
+    box-sizing: content-box;
+    font-weight: 400;
+    border: 1px solid #ececec;
+    width: calc(100% - 1px);
+  }
+  .Select-placeholder {
+    color: #d3d3d3;
+    padding: 0px 16px;
+  }
+  &.is-open {
+    .Select-control {
+      border: 1px solid #ececec;
+    }
+  }
+  &.is-focused {
+    &:not(.is-open):not(.is-invalid) {
+      .Select-control {
+        border-color: rgba(14, 151, 192, 0.5);
+        box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 1px rgba(14, 151, 192, 0.5);
+      }
+    }
+  }
+
+  &.is-invalid {
+    .Select-control {
+      border-color: #f4a29c;
+    }
+    .Select-menu-outer {
+      border-color: #f4a29c;
+    }
+    &.is-focused {
+      .Select-control {
+        border-color: #ed6157;
+        box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+      }
+      .Select-menu-outer {
+        border-color: #ed6157;
+        box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+      }
+    }
+  }
+}
```

Comment:
We should reuse or adapt the `form-control-state` mixin from overrides/forms.scss to handle these, and use the $brand-danger variable for color.